IdentityCore/src/cache/accessor/MSIDDefaultTokenCacheAccessor.m

```diff
@@ -659,11 +659,11 @@ - (BOOL)clearCacheForAccount:(MSIDAccountIdentifier *)accountIdentifier
             }
         }
         
-        [MSIDTelemetry stopCacheEvent:event withItem:nil success:result context:context];
+        CONDITIONAL_STOP_CACHE_EVENT(nil, result, context);
     }
     else
     {
-        [MSIDTelemetry stopCacheEvent:event withItem:nil success:YES context:context];
+        CONDITIONAL_STOP_CACHE_EVENT(nil, YES, context);
```

Comment:
Should event be provided as a parameter for clarity?

Reply:
Added, thanks!

Reply:
What I meant is could we pass event as a parameter into the macro?
e.g.         CONDITIONAL_STOP_CACHE_EVENT(event, nil, YES, context);

Reply:
For example, you could do something like this for clarity:
// Start event
id event = CONDITIONAL_START_EVENT()
// Stop event
CONDITIONAL_STOP_EVENT(event, item, YES, context)